coderd/workspacebuilds.go

```diff
@@ -327,6 +328,16 @@ func (api *API) postWorkspaceBuilds(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	previousWorkspaceBuild, err := api.Database.GetLatestWorkspaceBuildByWorkspaceID(ctx, workspace.ID)
```

Comment:
I'm not convinced if this is the right place to send the notification. We should send it after the successful update, so I would target the provisioner server?

Reply:
Good point. I think we can tweak this via wording.

For instance, this would be pretty flexible and let us keep the notification here:

> A new workspace build to update your workspace to template version 'boo' has been manually created/initiated.

Personally I think it's more valuable here as you know immediately this has been triggered, vs "maybe" sometime in the future _iff_ the build was successful. The update happens here whether or not the build is successful.

Reply:
> Personally I think it's more valuable here as you know immediately this has been triggered and since the update happens here whether or not the build is successful.

Fair 👍 A user could cancel the request.